File: CTDK_Python/092/_092.py
x = float(input("Nhap x: "))
n = int(input("Nhap n: "))

s = 1 - x
t = x
m = 1
dau = 1
i = 3
while(i <= 2 * n + 1):
    t = t * x * x
    m = m * i * (i - 1)
    s = s + dau * t / m
    i = i + 2
    dau = -dau

print("Tong la: ", s)

File: LTHAM/092/092.cpp
#include <iostream>
#include <cmath>
using namespace std;

float Tong(float, int);

int main()
{
	float x;
	cout << "Nhap x: ";
	cin >> x;

	int n;
	cout << "Nhap n: ";
	cin >> n;
	
	cout << "Tong la: " << Tong(x, n) << endl;
	return 0;
}

float Tong(float x, int n)
{
	float s = 1 - x;
	int m = 1;
	float t = x;
	int dau = 1;
	int i = 3;
	while (i <= 2 * n + 1)
	{
		m = m * (i - 1) * i;
		t = t * x * x;
		s = s + dau * t / m;
		dau = -dau;
		i += 2;
	}
	return s;
}